cineast-api/src/main/java/org/vitrivr/cineast/api/rest/handlers/actions/FindTagsByActionHandler.java

```diff
@@ -78,4 +84,41 @@
     public Class<IdList> inClass() {
         return IdList.class;
     }
+
+    @Override
+    public String getRoute() {
+        return "tags/by/id";
+    }
+
+    @Override
+    public String routeForGet() {
+        return String.format("find/tags/by/:%s/:%s", GET_PARAMETER_ATTRIBUTE, GET_PARAMETER_VALUE);
+    }
+
+    @Override
+    public String routeForPost() {
+        return "find/tags/by/id";
+    }
+
+    @Override
+    public String getDescription(RestHttpMethod method) {
+        switch(method) {
+        case GET:
+            return "Find tags by attribute and value";
+        case POST:
+            return "Find tags for given id list";
+        default:
+                return "Find tags by given id or attribute and value";
+        }
+    }
+
+    @Override
+    public Class<Tag> outClass() {
```

Comment:
See comment at FindTagsActionHandler regarding dedicated message for list of tags